test/golden/pretty_lines.golden

```diff
@@ -0,0 +1,31 @@
+a = "a"
+b = "bb"
+c = "cccc"
+d = "ddd"
+
+[baz]
+
+list =
+     [ "one"
```

Comment:
This is a strange indentation. In the rest of the file it uses 2 spaces, but here, it is 5 🤔 
I guess it should use the same indentation as the rest of the file and what is specified in the `PrintOptions` 🙂

Reply:
I took indentation from https://github.com/kowainik/tomland/issues/325#issuecomment-702650931
I thought it `[` should be under `=`
```
=
[
```
Ok, let it be 2 spaces. Probably default is 1 space, isn't it?

Reply:
Oh sorry, that was not an ideal example, as it had 3 letters 😄 
Thanks!